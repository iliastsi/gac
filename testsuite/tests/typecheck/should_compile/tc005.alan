unusedVar() : proc
  var1 : int;
  var2 : int;
  foo1() : proc
    var3 : int;
  {}
  foo2() : proc
    foo3() : proc
    { var1 = 42; }
  { foo1(); }
{}
