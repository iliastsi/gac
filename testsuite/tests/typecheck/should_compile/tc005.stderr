tc005.alan:3:3-6: Warning: Defined but not used: `var2'

tc005.alan:5:5-8: Warning: Defined but not used: `var3'

